app/modules/zoo-api/zooapi.config.js

```diff
@@ -2,8 +2,8 @@
 
 require('./zooapi.module.js')
     .constant('zooAPIConfig', {
-        slug: 'shakespeares-world-demo',
-
+        slug: 'simoneduca/shakespeares-world-demo',
+        projectId: '376',
```

Comment:
Just a side note: the `panoptes-javascript-client` always wants strings, and will error if you give it an integer